tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -495,36 +495,25 @@ def test_auto_insert_project_name_when_event_id_present_with_aggregate(self):
             {"project.name": self.project.slug, "id": "a" * 32, "count()": 1}
         ]
 
+    @override_options({"performance.issues.all.problem-creation": 1.0})
     def test_performance_issue_ids_filter(self):
-        project = self.create_project(name="hello")
-        _perf_group = self.create_group(
-            type=GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value,
-            project=project,
-            short_id=project.next_short_id(),
-        )
-
-        def hack_pull_out_data(jobs, projects):
-            _pull_out_data(jobs, projects)
-            for job in jobs:
-                job["event"].groups = [_perf_group]
-            return jobs, projects
-
-        data = load_data(
-            platform="transaction",
-            timestamp=before_now(minutes=10),
-            start_timestamp=before_now(minutes=11),
-        )
-        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
-            self.store_event(data=data, project_id=project.id)
+        with self.feature({"organizations:performance-issues-ingest": True}):
+            data = load_data(
+                platform="transaction",
+                timestamp=before_now(minutes=10),
+                start_timestamp=before_now(minutes=11),
+            )
+            data["fingerprint"] = [f"{GroupType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES.value}-group1"]
+            event = self.store_event(data=data, project_id=self.project.id)
```

Comment:
This is so much nicer! I wonder if it makes sense to pass `fingerprint` to load data?

Reply:
yup, I was gonna do that next. Just wanted to check first if the direction made sense 👍 It seems like it is, so I'm going to update `load_data` 🙏

Reply:
and I'll add some validation around fingerprint formatting since we derive the type from it as well